packages/hint-manifest-icons/src/hint.ts

```diff
@@ -302,6 +332,24 @@ export default class ManifestIconHint implements IHint {
             }
         };
 
+        const invalidManifest = ({ errors, resource }: ManifestInvalidSchema) => {
+            for (const error of errors) {
+                const isIconError = error.dataPath.match(iconsErrorRegex);
+
+                if (isIconError) {
+                    // Is error the right severity?
+                    const message = getMessage('iconPurposeInvalid', context.language, isIconError[1]);
+
+                    console.log(message);
```

Comment:
Is this console.log left over from debugging?

Reply:
yes! Good catch